go.mod

```diff
@@ -202,6 +200,8 @@ replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 // allows us to specify that as an option.
 replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
 
+replace github.com/lightningnetwork/lnd/kvdb => github.com/guggero/lnd/kvdb v1.2.2-0.20240315183436-acfe249b2d09
```

Comment:
This is making it very difficult for me to try and debug this further without having to reimplement your kvdb changes on top of 1.4.5

Reply:
If you look at the commit, that's just what's in #8555. I didn't push all tags to my fork, that's why the Golang pseudo version looks like 1.2.2.